pypdf/_writer.py

```diff
@@ -535,29 +591,7 @@ def insert_blank_page(
     def open_destination(
         self,
     ) -> Union[None, Destination, TextStringObject, ByteStringObject]:
-        """
-        Property to access the opening destination (``/OpenAction`` entry in
-        the PDF catalog). It returns ``None`` if the entry does not exist is not
-        set.
-
-        Raises:
-            Exception: If a destination is invalid.
-        """
-        if "/OpenAction" not in self._root_object:
-            return None
-        oa = self._root_object["/OpenAction"]
-        if isinstance(oa, (str, bytes)):
-            return create_string_object(str(oa))
-        elif isinstance(oa, ArrayObject):
-            try:
-                page, typ = oa[0:2]
-                array = oa[2:]
-                fit = Fit(typ, tuple(array))
-                return Destination("OpenAction", page, fit)
-            except Exception as exc:
-                raise Exception(f"Invalid Destination {oa}: {exc}")
-        else:
-            return None
+        return super().open_destination
```

Comment:
Do we really need to overwrite this method when we only have the `super` call anyway?

Reply:
Yes without this approach I've not been able to add the setter function (applicable only to PdfWriter)